src/System.Security.Cryptography.Csp/src/System/Security/Cryptography/CapiHelper.cs

```diff
@@ -698,7 +698,7 @@ private static void ValidateCspFlags(CspProviderFlags flags)
                                             CspProviderFlags.UseUserProtectedKey);
                 if ((flags & keyFlags) != CspProviderFlags.NoFlags)
                 {
-                    throw new ArgumentException(SR.Format(SR.Argument_InvalidValue, Convert.ToString(flags)));
+                    throw new ArgumentException(SR.Argument_InvalidValue, nameof(flags));
```

Comment:
Convert.ToString(flags) produces something like "UseExistingKey, CreateEphemeralKey" that could potentially be helpful. is it worth including it? It's too bad ArgumentExceptoin doesn't have a field for "actual value"